### ISSUE TITLE
Chapters

### ISSUE THREAD

**Issue by gjbex** [hpcugent/vsc_user_docs#16]
The Style Guide mentions that chapters should be in files with names starting with the chapter number, e.g., (ch2_...).
There are two issues with this:
1) If we go for this scheme, I'd suggest to use ch02_... since this makes viewing files in the shell easier, but,
2) this is an approach that is okay for now, because this document is mostly finished, but it is not so nice for future work.  Deciding to insert a chapter at some point will be a rather annoying operation.
So basically, I would forego the ch2_... prefix altogether.

I would also suggest to have a directory per chapter.  That leaves the option to split the TeX file into multiple files, should the need arise, and to make sure that auxiliary files have a place without cluttering the main directory.

Again, this is not so much with this document in mind, but rather for whenever we would start on something new.

**Comment by JensTimmerman** [comment]
we currently have `ch_` as a prefix for chapters.
An argument against chapter numbers in names is that chapter numbers are generated by latex, never explicitly defined by us, so adding a new chapter somewhere would mean we have to rename all files.

However, adding an ordering to the example directories does make sense, but this can be the order in which the examples should be followed, and does not have to be tied to the chapters in the textbook?
